.github/workflows/dotnet.yml

```diff
@@ -3,14 +3,13 @@ name: .NET
 on:
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
+  pull_request_review:
```

Comment:
Trigger the build on push to main or once a pull request review has been submitted

Reply:
What's your motivation for changing this? As a contributor, I'd expect the project to be built and tested when I opened a pull request. Ditto as a reviewer.